### ISSUE TITLE
audio_bitrate in file name

### ISSUE THREAD

**Issue by shahramsh** [inbasic/iaextractor#87]
Hi,
I always add [video_resolution] & [audio_bitrate] to the file name, so I know what I have downloaded,

but when I download 1080p videos, after which ffmpeg combines it with the audio (m4a),
the final combined version doesn't have the [audio_bitrate] anymore, it only has the video resolution.

**Comment by inbasic** [comment]
Right! This is a known bug. The reason is the combiner use video-only file name as the naming reference and this file does not have  `[audio_bitrate] ` in it!
